seqr_scripts/seqr_sample_qc.py

```diff
@@ -402,14 +402,12 @@ def main(args):
     mt = mt.annotate_cols(**pop_ht[mt.col_key])
 
     logger.info("Running Hail's sample qc...")
-    hail_metric_ht = run_hail_sample_qc(mt, data_type, data_source)
+    hail_metric_ht = run_hail_sample_qc(mt, data_type)
     mt = mt.annotate_cols(**hail_metric_ht[mt.col_key])
 
     logger.info("Exporting sample QC tables...")
     ht = mt.cols()
-    ht = ht.checkpoint(
-        sample_qc_ht_path(build, data_type, data_source, version, is_test), overwrite
-    )
+    # ht = ht.checkpoint(sample_qc_ht_path(build, data_type, data_source, version, is_test), overwrite)
```

Comment:
Purpose of leaving this as a commented line of code?

Reply:
Good catch, this should be uncommented!